### ISSUE TITLE
Manual install option to hide that EP is installed

### ISSUE THREAD

**Issue by jakimanboy** [valinet/ExplorerPatcher#1996]
Hi, is there is a way to manually install EP so that it can be hidden from work PC scan/checks on what program is installed? It is getting flagged as being a non-compliant app install so trying to see if there is a way to try to use EP without it getting flagged.

**Comment by pyrates999** [comment]
No.  I would not install it on your work computer anyways since administrative privileges for your account could be taken away at any time.
